build/layerutils/charts/helm_test.go

```diff
@@ -14,6 +14,12 @@
 
 package charts
 
+import (
+	"reflect"
+	"sort"
+	"testing"
+)
+
 /* func TestPackageHelmChart(t *testing.T) {
```

Comment:
Could you help remove the unused function？

Reply:
this function called in the case TestGetImageList, did  you mean TestRenderHelmChart and  TestPackageHelmChart?

Reply:
Yes, I think no one use these functions to do test. They are archived.

Reply:
ok.